Rel/latest

mmcshinsky-bitgo opened the pull request:
<!--
# Please be aware of the following when making your pull request:

## Description

Please include a summary of your proposed changes and which issue is being addressed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Issue Number

Internal Users - Please include the related internal tracking number (e.g. BG-000000).
External Users - Please link to any relevant github issues as necessary.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code compiles correctly for both Node and Browser environments
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My commits follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) and I have properly described any BREAKING CHANGES
- [ ] The ticket or github issue was included in the commit message as a reference
- [ ] I have made corresponding changes to the documentation and on any new/updated functions and/or methods - [jsdoc](https://jsdoc.app/)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
-->

mmcshinsky-bitgo committed:
chore(release): publish modules

 - @bitgo/abstract-eth@1.2.19
 - @bitgo/abstract-utxo@2.1.3
 - @bitgo/account-lib@4.5.1
 - bitgo@18.2.1
 - @bitgo/blockapis@1.6.4
 - @bitgo/express@9.26.0
 - @bitgo/key-card@0.1.5
 - @bitgo/sdk-api@1.10.1
 - @bitgo/sdk-coin-ada@2.3.14
 - @bitgo/sdk-coin-algo@1.3.19
 - @bitgo/sdk-coin-atom@3.1.2
 - @bitgo/sdk-coin-avaxc@2.4.10
 - @bitgo/sdk-coin-avaxp@3.5.10
 - @bitgo/sdk-coin-bch@1.3.19
 - @bitgo/sdk-coin-bcha@1.5.14
 - @bitgo/sdk-coin-bsc@3.5.1
 - @bitgo/sdk-coin-bsv@1.3.19
 - @bitgo/sdk-coin-btc@1.4.3
 - @bitgo/sdk-coin-btg@1.3.19
 - @bitgo/sdk-coin-celo@1.3.19
 - @bitgo/sdk-coin-cspr@1.2.19
 - @bitgo/sdk-coin-dash@1.3.19
 - @bitgo/sdk-coin-doge@1.8.14
 - @bitgo/sdk-coin-dot@2.2.8
 - @bitgo/sdk-coin-eos@1.3.19
 - @bitgo/sdk-coin-etc@1.2.19
 - @bitgo/sdk-coin-eth@4.4.1
 - @bitgo/sdk-coin-eth2@1.2.19
 - @bitgo/sdk-coin-ethw@1.3.19
 - @bitgo/sdk-coin-hbar@1.3.19
 - @bitgo/sdk-coin-ltc@2.0.14
 - @bitgo/sdk-coin-near@1.3.19
 - @bitgo/sdk-coin-polygon@1.5.3
 - @bitgo/sdk-coin-rbtc@1.3.19
 - @bitgo/sdk-coin-sol@2.4.3
 - @bitgo/sdk-coin-stx@2.0.3
 - @bitgo/sdk-coin-sui@3.0.3
 - @bitgo/sdk-coin-trx@1.7.3
 - @bitgo/sdk-coin-xlm@2.0.5
 - @bitgo/sdk-coin-xrp@1.4.1
 - @bitgo/sdk-coin-xtz@1.4.19
 - @bitgo/sdk-coin-zec@1.3.19
 - @bitgo/sdk-core@8.3.0
 - @bitgo/sdk-rpc-wrapper@1.1.3
 - @bitgo/sdk-test@1.2.19
 - @bitgo/sdk-unified-wallet@3.0.3
 - @bitgo/statics@15.1.1
 - @bitgo/unspents@0.13.2
 - @bitgo/utxo-bin@2.9.0
 - @bitgo/utxo-lib@8.1.0
 - @bitgo/utxo-ord@1.1.3
 - @bitgo/web-demo@1.7.0

mmcshinsky-bitgo committed:
feat(bitgo): dockerfile

BG-000000

mmcshinsky-bitgo committed:
Merge branch 'master' into rel/latest

mmcshinsky-bitgo committed:
chore(release): publish modules

 - @bitgo/abstract-eth@1.2.20
 - @bitgo/abstract-utxo@2.2.0
 - @bitgo/account-lib@4.5.2
 - bitgo@18.3.0
 - @bitgo/blockapis@1.6.5
 - @bitgo/express@9.26.1
 - @bitgo/key-card@0.1.6
 - @bitgo/sdk-api@1.10.2
 - @bitgo/sdk-coin-ada@2.3.15
 - @bitgo/sdk-coin-algo@1.3.20
 - @bitgo/sdk-coin-atom@3.1.3
 - @bitgo/sdk-coin-avaxc@2.4.11
 - @bitgo/sdk-coin-avaxp@3.5.11
 - @bitgo/sdk-coin-bch@1.3.20
 - @bitgo/sdk-coin-bcha@1.5.15
 - @bitgo/sdk-coin-bsc@3.5.2
 - @bitgo/sdk-coin-bsv@1.3.20
 - @bitgo/sdk-coin-btc@1.5.0
 - @bitgo/sdk-coin-btg@1.3.20
 - @bitgo/sdk-coin-celo@1.3.20
 - @bitgo/sdk-coin-cspr@1.2.20
 - @bitgo/sdk-coin-dash@1.3.20
 - @bitgo/sdk-coin-doge@1.8.15
 - @bitgo/sdk-coin-dot@2.2.9
 - @bitgo/sdk-coin-eos@1.3.20
 - @bitgo/sdk-coin-etc@1.2.20
 - @bitgo/sdk-coin-eth@4.4.2
 - @bitgo/sdk-coin-eth2@1.2.20
 - @bitgo/sdk-coin-ethw@1.3.20
 - @bitgo/sdk-coin-hbar@1.3.20
 - @bitgo/sdk-coin-ltc@2.0.15
 - @bitgo/sdk-coin-near@1.3.20
 - @bitgo/sdk-coin-polygon@1.5.4
 - @bitgo/sdk-coin-rbtc@1.3.20
 - @bitgo/sdk-coin-sol@2.4.4
 - @bitgo/sdk-coin-stx@2.0.4
 - @bitgo/sdk-coin-sui@3.0.4
 - @bitgo/sdk-coin-trx@1.7.4
 - @bitgo/sdk-coin-xlm@2.0.6
 - @bitgo/sdk-coin-xrp@1.4.2
 - @bitgo/sdk-coin-xtz@1.4.20
 - @bitgo/sdk-coin-zec@1.3.20
 - @bitgo/sdk-core@8.4.0
 - @bitgo/sdk-rpc-wrapper@1.1.4
 - @bitgo/sdk-test@1.2.20
 - @bitgo/sdk-unified-wallet@3.0.4
 - @bitgo/statics@15.1.2
 - @bitgo/unspents@0.14.0
 - @bitgo/utxo-bin@2.9.1
 - @bitgo/utxo-lib@8.2.0
 - @bitgo/utxo-ord@1.2.0
 - @bitgo/web-demo@1.7.1

mmcshinsky-bitgo committed:
feat(bitgo): dockerfile

BG-000000

mmcshinsky-bitgo committed:
Merge branch 'master' into rel/latest

mmcshinsky-bitgo committed:
chore(release): publish modules

 - @bitgo/abstract-eth@1.2.21
 - @bitgo/abstract-utxo@2.2.1
 - @bitgo/account-lib@4.5.3
 - bitgo@18.4.0
 - @bitgo/blockapis@1.6.6
 - @bitgo/express@9.26.2
 - @bitgo/key-card@0.1.7
 - @bitgo/sdk-api@1.10.3
 - @bitgo/sdk-coin-ada@2.3.16
 - @bitgo/sdk-coin-algo@1.3.21
 - @bitgo/sdk-coin-atom@3.1.4
 - @bitgo/sdk-coin-avaxc@2.4.12
 - @bitgo/sdk-coin-avaxp@3.5.12
 - @bitgo/sdk-coin-bch@1.3.21
 - @bitgo/sdk-coin-bcha@1.5.16
 - @bitgo/sdk-coin-bsc@3.5.3
 - @bitgo/sdk-coin-bsv@1.3.21
 - @bitgo/sdk-coin-btc@1.5.1
 - @bitgo/sdk-coin-btg@1.3.21
 - @bitgo/sdk-coin-celo@1.3.21
 - @bitgo/sdk-coin-cspr@1.2.21
 - @bitgo/sdk-coin-dash@1.3.21
 - @bitgo/sdk-coin-doge@1.8.16
 - @bitgo/sdk-coin-dot@2.2.10
 - @bitgo/sdk-coin-eos@1.3.21
 - @bitgo/sdk-coin-etc@1.2.21
 - @bitgo/sdk-coin-eth@4.4.3
 - @bitgo/sdk-coin-eth2@1.2.21
 - @bitgo/sdk-coin-ethw@1.3.21
 - @bitgo/sdk-coin-hbar@1.3.21
 - @bitgo/sdk-coin-ltc@2.0.16
 - @bitgo/sdk-coin-near@1.3.21
 - @bitgo/sdk-coin-polygon@1.5.5
 - @bitgo/sdk-coin-rbtc@1.3.21
 - @bitgo/sdk-coin-sol@2.4.5
 - @bitgo/sdk-coin-stx@2.0.5
 - @bitgo/sdk-coin-sui@3.0.5
 - @bitgo/sdk-coin-trx@1.7.5
 - @bitgo/sdk-coin-xlm@2.0.7
 - @bitgo/sdk-coin-xrp@1.5.0
 - @bitgo/sdk-coin-xtz@1.4.21
 - @bitgo/sdk-coin-zec@1.3.21
 - @bitgo/sdk-core@8.5.0
 - @bitgo/sdk-lib-mpc@8.3.0
 - @bitgo/sdk-rpc-wrapper@1.1.5
 - @bitgo/sdk-test@1.2.21
 - @bitgo/sdk-unified-wallet@3.0.5
 - @bitgo/unspents@0.14.1
 - @bitgo/utxo-bin@2.9.2
 - @bitgo/utxo-lib@8.3.0
 - @bitgo/utxo-ord@1.2.1
 - @bitgo/web-demo@1.7.2

mmcshinsky-bitgo committed:
feat(bitgo): dockerfile

BG-000000

sachushaji reviewed (commented):
Unit test seems to be failing

LeoKotschenreuther commented on modules/abstract-utxo/package.json:
If we are going from version 2.1.2 and adding features (but no breaking changes), shouldn't the new version be 2.2.0?

LeoKotschenreuther reviewed (dismissed):
I got confused about the versioning but this is not blocking.
Did we do a few releases since last changing our package.json files that could explain us skipping a few versions?

aarvay reviewed (commented):
flushing from queue. required build step failing.

mmcshinsky-bitgo committed:
Merge branch 'master' into rel/latest

mmcshinsky-bitgo committed:
chore(release): publish modules

 - @bitgo/abstract-eth@1.2.22
 - @bitgo/abstract-utxo@2.3.0
 - @bitgo/account-lib@4.6.0
 - bitgo@18.5.0
 - @bitgo/blockapis@1.6.7
 - @bitgo/express@9.26.3
 - @bitgo/key-card@0.1.8
 - @bitgo/sdk-api@1.10.4
 - @bitgo/sdk-coin-ada@2.3.17
 - @bitgo/sdk-coin-algo@1.3.22
 - @bitgo/sdk-coin-atom@3.2.0
 - @bitgo/sdk-coin-avaxc@2.4.13
 - @bitgo/sdk-coin-avaxp@3.5.13
 - @bitgo/sdk-coin-bch@1.3.22
 - @bitgo/sdk-coin-bcha@1.5.17
 - @bitgo/sdk-coin-bsc@3.5.4
 - @bitgo/sdk-coin-bsv@1.3.22
 - @bitgo/sdk-coin-btc@1.5.2
 - @bitgo/sdk-coin-btg@1.3.22
 - @bitgo/sdk-coin-celo@1.3.22
 - @bitgo/sdk-coin-cspr@1.2.22
 - @bitgo/sdk-coin-dash@1.3.22
 - @bitgo/sdk-coin-doge@1.9.0
 - @bitgo/sdk-coin-dot@2.3.0
 - @bitgo/sdk-coin-eos@1.3.22
 - @bitgo/sdk-coin-etc@1.2.22
 - @bitgo/sdk-coin-eth@4.4.4
 - @bitgo/sdk-coin-eth2@1.2.22
 - @bitgo/sdk-coin-ethw@1.3.22
 - @bitgo/sdk-coin-hbar@1.3.22
 - @bitgo/sdk-coin-ltc@2.0.17
 - @bitgo/sdk-coin-near@1.4.0
 - @bitgo/sdk-coin-polygon@1.5.6
 - @bitgo/sdk-coin-rbtc@1.3.22
 - @bitgo/sdk-coin-sol@2.5.0
 - @bitgo/sdk-coin-stx@2.0.6
 - @bitgo/sdk-coin-sui@3.0.6
 - @bitgo/sdk-coin-trx@1.7.6
 - @bitgo/sdk-coin-xlm@2.0.8
 - @bitgo/sdk-coin-xrp@1.5.1
 - @bitgo/sdk-coin-xtz@1.4.22
 - @bitgo/sdk-coin-zec@1.3.22
 - @bitgo/sdk-core@8.6.0
 - @bitgo/sdk-lib-mpc@8.3.1
 - @bitgo/sdk-rpc-wrapper@1.1.6
 - @bitgo/sdk-test@1.2.22
 - @bitgo/sdk-unified-wallet@3.0.6
 - @bitgo/statics@16.0.0
 - @bitgo/unspents@0.14.2
 - @bitgo/utxo-bin@2.9.3
 - @bitgo/utxo-lib@8.4.0
 - @bitgo/utxo-ord@1.2.2
 - @bitgo/web-demo@1.7.3

mmcshinsky-bitgo committed:
feat(bitgo): dockerfile

BG-000000

mmcshinsky-bitgo commented on modules/abstract-utxo/package.json:
I'd have to look at all the commits. This PR has a couple stable releases in it, so 2.2.0 was probably farther back in the commit history

mmcshinsky-bitgo committed:
Merge branch 'master' into rel/latest

mmcshinsky-bitgo committed:
chore(release): publish modules

 - @bitgo/abstract-cosmos@1.1.0
 - @bitgo/abstract-eth@1.2.23
 - @bitgo/abstract-utxo@3.0.0
 - @bitgo/account-lib@4.7.0
 - bitgo@19.0.0
 - @bitgo/blockapis@1.6.8
 - @bitgo/express@9.27.0
 - @bitgo/key-card@0.1.9
 - @bitgo/sdk-api@1.10.5
 - @bitgo/sdk-coin-ada@2.3.18
 - @bitgo/sdk-coin-algo@1.3.23
 - @bitgo/sdk-coin-atom@3.3.0
 - @bitgo/sdk-coin-avaxc@2.4.14
 - @bitgo/sdk-coin-avaxp@3.5.14
 - @bitgo/sdk-coin-bch@1.3.23
 - @bitgo/sdk-coin-bcha@1.5.18
 - @bitgo/sdk-coin-bsc@3.6.0
 - @bitgo/sdk-coin-bsv@1.3.23
 - @bitgo/sdk-coin-btc@1.5.3
 - @bitgo/sdk-coin-btg@1.3.23
 - @bitgo/sdk-coin-celo@1.3.23
 - @bitgo/sdk-coin-cspr@1.2.23
 - @bitgo/sdk-coin-dash@1.3.23
 - @bitgo/sdk-coin-doge@1.9.1
 - @bitgo/sdk-coin-dot@2.3.1
 - @bitgo/sdk-coin-eos@1.3.23
 - @bitgo/sdk-coin-etc@1.2.23
 - @bitgo/sdk-coin-eth@4.5.0
 - @bitgo/sdk-coin-eth2@1.2.23
 - @bitgo/sdk-coin-ethw@1.3.23
 - @bitgo/sdk-coin-hbar@1.3.23
 - @bitgo/sdk-coin-ltc@2.0.18
 - @bitgo/sdk-coin-near@1.4.1
 - @bitgo/sdk-coin-osmo@1.1.0
 - @bitgo/sdk-coin-polygon@1.5.7
 - @bitgo/sdk-coin-rbtc@1.3.23
 - @bitgo/sdk-coin-sol@2.5.1
 - @bitgo/sdk-coin-stx@2.0.7
 - @bitgo/sdk-coin-sui@3.0.7
 - @bitgo/sdk-coin-trx@1.8.0
 - @bitgo/sdk-coin-xlm@2.0.9
 - @bitgo/sdk-coin-xrp@1.5.2
 - @bitgo/sdk-coin-xtz@1.4.23
 - @bitgo/sdk-coin-zec@1.3.23
 - @bitgo/sdk-core@8.7.0
 - @bitgo/sdk-lib-mpc@8.4.0
 - @bitgo/sdk-rpc-wrapper@1.1.7
 - @bitgo/sdk-test@1.2.23
 - @bitgo/sdk-unified-wallet@3.0.7
 - @bitgo/statics@16.1.0
 - @bitgo/unspents@0.15.0
 - @bitgo/utxo-bin@2.9.4
 - @bitgo/utxo-lib@9.0.0
 - @bitgo/utxo-ord@1.2.3
 - @bitgo/web-demo@1.7.4

mmcshinsky-bitgo committed:
feat(bitgo): update docker file

BG-000000

mmcshinsky-bitgo committed:
fix(sdk-coin-bsc): remove unused dependency

BG-000000
